src/align.cpp

```diff
@@ -654,7 +655,7 @@ void align_preprocessor(void)
 } // align_preprocessor
 
 
-chunk_t *align_assign(chunk_t *first, size_t span, size_t thresh)
+chunk_t *align_assign(chunk_t *first, size_t span, size_t thresh, size_t *p_nl_count)
 {
```

Comment:
It is for storing the number of new lines found in a sub-level.  The first level gets NULL since it has no parent level.  Then &sub_nl_count is passed to the sub-level's (recursive) call to align_assign.  When it returns, this value is passed to AlignStack::NewLines functions to help determine if the configured span has been exceeded.

I ported this functionality from the existing align_var_def_brace function.